src/corecel/sys/ScopedProfiling.hip.cc

```diff
@@ -0,0 +1,85 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2023 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file corecel/sys/ScopedProfiling.hip.cc
+//! \brief The roctx implementation of \c ScopedProfiling
+//---------------------------------------------------------------------------//
+
+#include "ScopedProfiling.hh"
+
+#include <roctracer/roctx.h>
+
+#include "corecel/io/Logger.hh"
+
+#include "celeritas_sys_config.h"
```

Comment:
Add this to the top-level `.clang-format` include sorting oder regex

Reply:
This is already included in the regex no? There is the following rule:
```
  - Regex:           '^"celeritas_[a-z_]+\.h"'
 ```

Reply:
Oh ok, that's right. I thought from the output that this was out of order.